pulsar-functions/instance/src/main/python/python_instance.py

```diff
@@ -460,3 +481,45 @@ def close(self):
 
     if self.pulsar_client:
       self.pulsar_client.close()
+
+  def get_schema(self, schema_type, type_class_name, schema_properties):
+    schema = DEFAULT_SCHEMA
+    if schema_type == "" or schema_type is None:
+      schema = DEFAULT_SCHEMA
+    elif schema_type.lower() == "string":
+      schema = pulsar.schema.StringSchema()
+    elif schema_type.lower() == "json":
+      record_kclass = self.get_record_class(type_class_name)
+      schema = pulsar.schema.JsonSchema(record_kclass)
+    elif schema_type.lower() == "avro":
+      record_kclass = self.get_record_class(type_class_name)
+      schema = pulsar.schema.AvroSchema(record_kclass, schema_properties)
+    else:  # load custom schema
```

Comment:
how about other schema types like PROTOBUF, PROTOBUF_NATIVE, and KeyValue?

Reply:
python client only supports avro, json, string and bytes schema for now

I will add todo here